### PR TITLE
Parse to number string for all balances instead of hex

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/sync/index.js [hunk 1]
```diff
@@ -555,7 +556,8 @@ async function updateOracleBalance(oracleAddress, topicSet, db) {
     }
   }
 
-  const balances = _.map(value[0].slice(0, oracle.numOfResults), balanceBN => balanceBN.toJSON());
+  const balances = _.map(value[0].slice(0, oracle.numOfResults),
+    balanceBN => Web3Utils.hexToNumberString(balanceBN.toJSON()));
```
Comment (frankobe) [inline src/sync/index.js]: should we use `toString` instead of `toJSON` for all

Comment (dwalintukan) [reply src/sync/index.js]: Why not? toJSON is an alias for toString(16)

Comment (frankobe) [reply src/sync/index.js]: why not directly use toString(10)?

Comment (dwalintukan) [reply src/sync/index.js]: oh right lol good point. will change.

**Pull request merged**